Sources/Transaction/Memos/DestinationMemoDataUtils.swift

```diff
@@ -0,0 +1,186 @@
+//
```

Comment:
Filename is different than classname... has "DataUtils" suffix where the other memo util classes are just <memotype>Utils

Reply:
updated